tests/e2e/utils/src/flows/shopper.js

```diff
@@ -127,6 +129,33 @@ const shopper = {
 		await removeButton.click();
 	},
 
+	emptyCart: async () => {
+		await page.goto( SHOP_CART_PAGE, {
+			waitUntil: 'networkidle0',
+		} );
+
+		// Remove products if they exist
+		if ( await page.$( '.remove' ) !== null ) {
+			products = await page.$( '.remove' );
+			while ( products.length > 0 ) {
+				for (let p = 0; p < products.length; p++ ) {
+					await page.click( p );
+					await uiUnblocked();
+				}
+				products = await page.$( '.remove' );
+			}
+		}
+
+		// Remove coupons if they exist
+		if ( await page.$( '.woocommerce-remove-coupon' ) !== null ) {
+			await page.click( '.woocommerce-remove-coupon' );
+			await uiUnblocked()
```

Comment:
Nitpick- this is missing the trailing semi-colon.

Reply:
Ack, good catch. Fixed!